lib/modules/manager/gradle-wrapper/artifacts.ts

```diff
@@ -194,6 +194,48 @@ export async function updateArtifacts({
       );
     }
 
+    let writeLocksCmd = await prepareGradleCommand(gradlewFile);
+    if (!writeLocksCmd) {
+      logger.info('No gradlew found - skipping Artifacts update');
+      return null;
+    }
+    writeLocksCmd += ' :wrapper --write-locks';
```

Comment:
This ignores gradle subprojects

Reply:
Can you elaborate? AFAIK the wrapper task is only present on the root project. The difference between `:wrapper` and `wrapper` is semantics, and since Renovate was already using `:wrapper` I didn't want to change that, even though the convention is to use `wrapper`.